### PR TITLE
Better Port Conflict Error

### PR DESCRIPTION
**Pull request by arknable**
1. Check whether specified port is being used by other application
2. If being used by other application, check the PID of that application
3. Display human-readable message with PID information from no. 2.

### CODE REVIEW

File: cmd/cmd_pkg.go [hunk 1]
```diff
@@ -219,9 +223,53 @@ func loadCommandHandler(cmd *cobra.Command, args []string) {
 	buildImageFlags := NewBuildImageCommandFlags(flags)
 	runLocalInstanceFlags := NewRunLocalInstanceCommandFlags(flags)
 	pkgFlags := NewPkgCommandFlags(flags)
-
 	pkgFlags.Package = args[0]
 
+	for _, port := range runLocalInstanceFlags.Ports {
```
Comment (fabioDMFerreira) [inline cmd/cmd_pkg.go:228]: Let's do this in `flags_run_local_instance` to cover all the cases where the instances are launched locally (https://github.com/nanovms/ops/blob/master/cmd/flags_run_local_instance.go#L35).

File: cmd/cmd_pkg.go [hunk 2]
```diff
@@ -219,9 +223,53 @@ func loadCommandHandler(cmd *cobra.Command, args []string) {
...
+
+		if conn != nil {
+			conn.Close()
+
+			var (
+				pid     string
+				message = fmt.Sprintf("Port %v is being used by other application", port)
+			)
+
+			if runtime.GOOS == "linux" || runtime.GOOS == "darwin" {
```
Comment (fabioDMFerreira) [inline cmd/cmd_pkg.go:242]: I'd prefer to keep architecture-specific code in separated files like `lepton/ldd_linux.go` or `lepton/ldd_darwin`.

**Review by fabioDMFerreira — CHANGES_REQUESTED** [review]

File: cmd/cmd_pkg.go (hunk 1 above)
Comment (arknable) [reply cmd/cmd_pkg.go]: Fixed by https://github.com/nanovms/ops/pull/1025/commits/991df4ea689ffaeab24358c7c00c80f62c20f7b2

File: cmd/cmd_pkg.go (hunk 2 above)
Comment (arknable) [reply cmd/cmd_pkg.go]: Fixed by https://github.com/nanovms/ops/pull/1025/commits/991df4ea689ffaeab24358c7c00c80f62c20f7b2

**Comment by arknable** [comment]
@fabioDMFerreira Please review

**Comment by fabioDMFerreira** [comment]
@arknable please rebase

**Comment by arknable** [comment]
@fabioDMFerreira rebased

**Comment by fabioDMFerreira** [comment]
@arknable I'm still seeing old commits in this PR

**Comment by arknable** [comment]
> @arknable I'm still seeing old commits in this PR

oh you right. checking

**Comment by arknable** [comment]
@fabioDMFerreira looks like my gitkraken is causing double commits upon rebase. I have created [New PR](https://github.com/nanovms/ops/pull/1033) with correct commit histories.

**Comment by arknable** [comment]
Replaced with [This PR](https://github.com/nanovms/ops/pull/1033) due to git client issue.

**Pull request closed without merging**